server/protocol/qdatastreamprotocol.py

```diff
@@ -151,7 +152,7 @@ def close(self):
             await self.writer.drain()
 
     async def send_raw(self, data):
-        server.stats.incr('server.sent_messages')
+        metrics.sent_messages.labels("raw").inc()
```

Comment:
I don't think there's a reason to distinguish between these two. Especially since in my asyncio fix PR I'm refactoring `send_message` to call `send_raw`.